study-builder/tenants/register-user-in-pepper.js

```diff
@@ -243,7 +257,7 @@ function (user, context, callback) {
                     // all is well
                     var ddpUserGuid = body.ddpUserGuid;
                     user.app_metadata.user_guid = ddpUserGuid;
-
+                    //todo looks like updating on every re-login .. check and avoid the update when not needed
```

Comment:
The update itself is not required, but injecting the guid into the token claims is required for every run of this token (see line 263 below).